File: Serene3/Serene3.Web/Modules/com/order/orderColumns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace Serene3.com.Columns;

[ColumnsScript("com.order")]
[BasedOnRow(typeof(orderRow), CheckNames = true)]
public class orderColumns
{
    [EditLink]
    public string Userid { get; set; }
    public string Name { get; set; }
    public string Phone { get; set; }
    public string Email { get; set; }
 
    public bool Onlinepaid { get; set; }
    public DateOnly EntityDate { get; set; }
    public string CustomerName { get; set; }
    public string CountryName { get; set; }
    public string StateName { get; set; }
    public string CityName { get; set; }
    public string Address { get; set; }
    public DateTime OrderDate { get; set; }
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int Id { get; set; }
    public int Productid { get; set; }
}

File: Serene3/Serene3.Web/Modules/ServerTypes/com/orderColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { orderRow } from "./orderRow";

export interface orderColumns {
    Userid: Column<orderRow>;
    Name: Column<orderRow>;
    Phone: Column<orderRow>;
    Email: Column<orderRow>;
    Onlinepaid: Column<orderRow>;
    EntityDate: Column<orderRow>;
    CustomerName: Column<orderRow>;
    CountryName: Column<orderRow>;
    StateName: Column<orderRow>;
    CityName: Column<orderRow>;
    Address: Column<orderRow>;
    OrderDate: Column<orderRow>;
    Id: Column<orderRow>;
    Productid: Column<orderRow>;
}

export class orderColumns extends ColumnsBase<orderRow> {
    static readonly columnsKey = 'com.order';
    static readonly Fields = fieldsProxy<orderColumns>();
}